system-int-test/test/cmr/system_int_test/search/granule_shapefile_search_test.clj

```diff
@@ -32,6 +33,99 @@
   {"ESRI" {:extension "zip" :mime-type mt/shapefile}
    "GeoJSON" {:extension "geojson" :mime-type mt/geojson}
    "KML" {:extension "kml" :mime-type mt/kml}})
+
+(deftest granule-shapefile-failure-cases
+  (side/eval-form `(shapefile/set-enable-shapefile-parameter-flag! true))
+  (let [saved-shapefile-max-size (shapefile-middleware/max-shapefile-size)
+        _ (side/eval-form `(shapefile-middleware/set-max-shapefile-size! 50000))
```

Comment:
nit: move these to a fixture. I can't remember if your revert to default code below will get ran on an exception.

Reply:
I only want these to run for the first set of tests (the failure cases), whereas I believe `use-fixtures` applies to the whole file. Also, exceptions in the tests do not prevent the revert to default.